app/models/conversion_host.rb

```diff
@@ -454,7 +454,7 @@ def ansible_playbook(playbook, extra_vars = {}, miq_task_id = nil, auth_type = '
       raise MiqException::MiqInvalidCredentialsError, _("Unknown auth type: %{auth_type}") % {:auth_type => auth.authtype}
     end
 
-    params << {:extra_vars => "'#{extra_vars.to_json}'"}
+    extra_vars.each { |k, v| params << {:extra_vars= => "#{k}='#{v}'"} }
```

Comment:
I'm going to merge this for now since it's working for you, but I still think it's wrong.  For example, it manually embeds single quotes around the value without taking care of whether single quotes are actually in the strings being embedded.  However it was like that previously as well, so it's no different with respect to this PR.

Reply:
Thanks. When we move to AnsibleRunner, I guess this won't be a topic anymore.